Automatic masks range selection for better visualization

TwistedW opened the pull request:
The range of the mask needs to be changed.

ksofiyuk commented:
Hi, it's a good idea! I updated your code as there was missed non-None check for the mask before multiplication. And I also replaced `255` multiplier with `255 // mask.max()`, which also better handles masks with multiple objects.

ksofiyuk commented:
So with this fix, saved masks become more visually distinguishable.

For 1 object:
![mask1](https://user-images.githubusercontent.com/5459767/87776109-e3421b80-c82f-11ea-84f4-3dc822835a16.png)

For 4 objects:
![mask4](https://user-images.githubusercontent.com/5459767/87776113-e50bdf00-c82f-11ea-8bba-547c5fa045c7.png)

TwistedW commented:
> Hi, it's a good idea! I updated your code as there was missed non-None check for the mask before multiplication. And I also replaced `255` multiplier with `255 // mask.max()`, which also better handles masks with multiple objects.

Well, a better way.
